From: Tom Herbers <mail@tomherbers.de>
Date: Mon, 2 Oct 2023 17:00:58 +0200
Subject: Revert "mt76: update to the latest version from the 22.03 branch"

This reverts commit 8da4e8fb5687c93091c3759fb983bc59c6312683.

Neccesary because of https://github.com/freifunk-gluon/gluon/issues/3003

diff --git a/package/kernel/mt76/Makefile b/package/kernel/mt76/Makefile
index 3deeed831589c97f0eee6898dc1ade9644468004..9b10fc3e6fa79a87c14ad3287d3fd155fb63a806 100644
--- a/package/kernel/mt76/Makefile
+++ b/package/kernel/mt76/Makefile
@@ -8,9 +8,9 @@ PKG_LICENSE_FILES:=
 
 PKG_SOURCE_URL:=https://github.com/openwrt/mt76
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_DATE:=2023-09-11
-PKG_SOURCE_VERSION:=bdf8ea71700746c634c064d6477aa143f821d6f6
-PKG_MIRROR_HASH:=a080a83068abc204d1f563022391625875ef77deda14be49888c179ff78fa8c8
+PKG_SOURCE_DATE:=2023-08-26
+PKG_SOURCE_VERSION:=de38fe7d4cb365312aa570cf37a4c0eff6e15650
+PKG_MIRROR_HASH:=d8b86332060668b36698e3b3703b8735003c09d7dc612327afa9272eead5238d
 
 PKG_MAINTAINER:=Felix Fietkau <nbd@nbd.name>
 PKG_USE_NINJA:=0
